### DISCUSSION TITLE
Outdated Tutorial

### DISCUSSION THREAD

**Discussion by ljfreelancer88** [CoderLine/alphaTab#1167] [General]
I noticed that the player on the homepage is updated. But when I check the tutorial on this link https://alphatab.net/docs/tutorial-web/player, the player is not consistent. Is it okay, to update the code as well. Thank you in advance.

**Comment by Danielku15** [comment]
I fear I don't understand this request clearly. The player shown on the main homepage is a lot more advanced. The tutorial only shows some very first steps towards such a player but not everything. It is on purpose that not exactly the same player is build. The tutorial is meant to teach developers how to get started using alphaTab for their own purposes and then use the overall API reference and documentation to build something specific for their needs of their website or app.
